docker-compose.yaml

```diff
@@ -19,3 +19,26 @@ services:
     command: 'pserve c2c://${INI_FILE}'
     ports:
       - 8080:6543
+
+  minio:
```

Comment:
Should I declare a volume for 'minio' or isn't it necessary since we don't need a mapping to the host?